.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,103 @@
+{
+  "name": "SmartOrder Container",
+  "service": "smart-order",
+  "runServices": ["smart-order"],
+  "dockerComposeFile": "docker-compose.yml",
+  "workspaceFolder": "/workspaces/SmartOrder",
+  "onCreateCommand": "/bin/bash .devcontainer/bootstrap.sh",
+  "containerEnv": {
+    "VSCODE": "true"
+  },
+  "mounts": [
+    "source=${localWorkspaceFolder}/../,target=/workspaces,type=bind,consistency=cached"
```

Comment:
Is this mounting whatever folder contains all your projects?

Reply:
That is the goal, yes.  I figure that the parent folder of the `SmartOrder` project would the one.  Alternatively, we could have a different repo host them all, and this stuff would just sit in the top of it.  The other 3 could be subrepositories